flink-python/pyflink/table/tests/test_table_environment_api.py

```diff
@@ -407,7 +408,7 @@ def test_from_data_stream_atomic(self):
         self.assertEqual("""(
   `f0` RAW('[B', '...')
 )""",
-                         result._j_table_result.getResolvedSchema().toString())
+                         str(result.get_resolved_schema()))
```

Comment:
why was this necessary?

Reply:
Wasn't necessary - I just thought it would make sense to change this test to use the new Python-side equivalents of these functions.